srcpkgs/novprog/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'novprog'
+pkgname=novprog
+version=3.1.7
+revision=1
+build_style=qmake
+hostmakedepends="pkg-config qt5-qmake"
+makedepends="qt5-widgets qt5-tools-devel"
+short_desc="Wordcount tracking program"
+maintainer="Orphaned <orphan@voidlinux.org>"
```

Comment:
It's usually expected that if you're making a PR for a new package, you will also maintain it. Keeping it up to date and potentially taking a look at issues people have, if any.

Reply:
Actually its required.  Packages are not accepted pre-orphaned.

Reply:
Sure, I can do that. I'll add my contact info then.